sdk/eventhub/azure-eventhub-checkpointstoretable/tests/test_storage_table_partition_manager.py

```diff
@@ -13,14 +13,15 @@
 from azure.eventhub.extensions.checkpointstoretable import TableCheckpointStore
 from azure.eventhub.exceptions import OwnershipLostError
 
-STORAGE_CONN_STR = [
-    #os.environ.get("AZURE_STORAGE_CONN_STR", "Azure Storage Connection String"),
-    os.environ.get("AZURE_COSMOS_CONN_STR", "Azure Storage Connection String"),
+STORAGE_ENV_KEYS = [
+    "AZURE_TABLES_CONN_STR",
+    "AZURE_COSMOS_CONN_STR"
 ]
 
 
-def get_live_storage_table_client(storage_connection_str):
+def get_live_storage_table_client(conn_str_env_key):
```

Comment:
do we need `@pytest.mark.parametrize("conn_str_env_key", STORAGE_ENV_KEYS)` on the top?

Reply:
I don't believe so since this is just a helper method, which is called by the three live tests at the bottom that all have `pytest.mark.parametrize`